diff --git a/dist/turbo.es2017-esm.js b/dist/turbo.es2017-esm.js
index cfe4084..fd54933 100644
--- a/dist/turbo.es2017-esm.js
+++ b/dist/turbo.es2017-esm.js
@@ -802,6 +802,7 @@ class FormSubmission {
         var _a;
         this.state = FormSubmissionState.stopped;
         (_a = this.submitter) === null || _a === void 0 ? void 0 : _a.removeAttribute("disabled");
+        (_a = this.submitter) === null || _a === void 0 ? void 0 : _a.focus();
         this.resetSubmitterText();
         dispatch("turbo:submit-end", {
             target: this.formElement,
diff --git a/dist/turbo.es2017-umd.js b/dist/turbo.es2017-umd.js
index 303ecf2..08dd969 100644
--- a/dist/turbo.es2017-umd.js
+++ b/dist/turbo.es2017-umd.js
@@ -808,6 +808,7 @@ Copyright © 2023 37signals LLC
             var _a;
             this.state = FormSubmissionState.stopped;
             (_a = this.submitter) === null || _a === void 0 ? void 0 : _a.removeAttribute("disabled");
+            (_a = this.submitter) === null || _a === void 0 ? void 0 : _a.focus();
             this.resetSubmitterText();
             dispatch("turbo:submit-end", {
                 target: this.formElement,
