diff --git a/node_modules/@hotwired/turbo/dist/turbo.es2017-esm.js b/node_modules/@hotwired/turbo/dist/turbo.es2017-esm.js
index 963422f..7820263 100644
--- a/node_modules/@hotwired/turbo/dist/turbo.es2017-esm.js
+++ b/node_modules/@hotwired/turbo/dist/turbo.es2017-esm.js
@@ -551,7 +551,8 @@ class StreamMessage {
         }, []);
     }
     get templateChildren() {
-        return Array.from(this.templateElement.content.children);
+        const content = this.templateElement.content;
+        return content.children ? Array.from(content.children) : Array.from(content.childNodes).filter((tag) => tag.tagName);
     }
 }
 StreamMessage.contentType = "text/vnd.turbo-stream.html";
diff --git a/node_modules/@hotwired/turbo/dist/turbo.es2017-umd.js b/node_modules/@hotwired/turbo/dist/turbo.es2017-umd.js
index 101db1f..a63cfbe 100644
--- a/node_modules/@hotwired/turbo/dist/turbo.es2017-umd.js
+++ b/node_modules/@hotwired/turbo/dist/turbo.es2017-umd.js
@@ -557,7 +557,8 @@ Copyright © 2021 Basecamp, LLC
             }, []);
         }
         get templateChildren() {
-            return Array.from(this.templateElement.content.children);
+            const content = this.templateElement.content;
+            return content.children ? Array.from(content.children) : Array.from(content.childNodes).filter((tag) => tag.tagName);
         }
     }
     StreamMessage.contentType = "text/vnd.turbo-stream.html";
